matching-web/src/main/java/com/graphhopper/matching/http/MatchServlet.java

```diff
@@ -124,13 +124,13 @@ public void doPost(HttpServletRequest httpReq, HttpServletResponse httpRes)
                 matchRsp = matching.doWork(gpxFile.getEntries());
 
                 // fill GHResponse for identical structure            
-                Path path = matching.calcPath(matchRsp);
-                Translation tr = trMap.getWithFallBack(locale);
-                DouglasPeucker peucker = new DouglasPeucker().setMaxDistance(wayPointMaxDistance);
-                PathMerger pathMerger = new PathMerger().
-                        setDouglasPeucker(peucker).
-                        setSimplifyResponse(wayPointMaxDistance > 0);
-                pathMerger.doWork(matchGHRsp, Collections.singletonList(path), tr);
+//                Path path = matching.calcPath(matchRsp);
+//                Translation tr = trMap.getWithFallBack(locale);
+//                DouglasPeucker peucker = new DouglasPeucker().setMaxDistance(wayPointMaxDistance);
+//                PathMerger pathMerger = new PathMerger().
+//                        setDouglasPeucker(peucker).
+//                        setSimplifyResponse(wayPointMaxDistance > 0);
+//                pathMerger.doWork(matchGHRsp, Collections.singletonList(path), tr);
```

Comment:
As I didn't know how to get it working - I'll update that when I get the web stuff working.